src/types.ts

```diff
@@ -70,6 +70,7 @@ export interface OptionContract {
 }
 
 export interface OptionOrderParams extends OptionContract {
+    payPremium?: boolean | null,
```

Comment:
Can you add a comment to explain what this is, just like how the rest of the variables have?